tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -781,6 +781,160 @@ int main(int argc, char **argv)
             }
         }
 
+        /* Test equal preference grouping */
+        {
+            /* Test server cipher preference negotiation for client-preferenced equal preference group */
+            {
+                EXPECT_SUCCESS(s2n_enable_tls13_in_test());
+                
+                /**
+                 * TODO: This cipher suite will be moved to s2n_cipher_preferences w/ it's own security policy.
+                 *       For now, it's only used for testing purposes.
+                 * Cipher suite w/ all TLS cipher suites in one equal preference group.
+                 */
+                static struct s2n_cipher_suite *s2n_all_tls13_cipher_suites_equal_preference[] = {
```

Comment:
If we've got particularly complex set-up code for a test, we usually put it outside of the test bracket. So for your test here I would organize it like: 
```
{
    // Set up specific security policy needed for below test(lines 790-814 currently)
    {
        // Actual test code
    }
}
```
This helps us focus on what you're actually testing

Reply:
Ok